Modules/vc-module-orders/VirtoCommerce.OrdersModule.Web/Security/OrderReadPricesPermission.cs

```diff
@@ -0,0 +1,31 @@
+using VirtoCommerce.Platform.Core.Security;
+
+namespace VirtoCommerce.OrderModule.Web.Security
+{
+    public static class OrderReadPricesPermission
+    {
+        public static string ApplyResponseGroupFiltering(Permission[] permissions, string respGroup)
+        {
+            var result = respGroup;
+            //TODO
```

Comment:
Why cannot uncomment it? As i know, permissions should work in v3, maybe scopes are not fully supported yet.